server/monitor.go

```diff
@@ -168,7 +172,11 @@ func (s *Server) HandleConnz(w http.ResponseWriter, r *http.Request) {
 		ci.OutMsgs = client.outMsgs
 		ci.OutBytes = client.outBytes
 		ci.NumSubs = client.subs.Count()
-		ci.Pending = client.bw.Buffered()
+		if client.internal {
+			ci.Pending = 0
```

Comment:
Maybe assignment here can be omitted since it is the zero-initialized value already?